### PR TITLE
Assignmnets on loose coupling

### PR DESCRIPTION
**Pull request by kevinsanthosh**

### CODE REVIEW

File: opt-may-2016/kevin/com/two95/assignment5/Car.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package com.two95.assignment5;
+
+public abstract class Car {
+	private String name;
+	private String colour;
+	private int enginePower;
+	private double milage;
+	public abstract void designEngine();
```
Comment (saba83rish) [inline opt-may-2016/kevin/com/two95/assignment5/Car.java:8]: this is an overkill you dont need it..but its not wrong

File: opt-may-2016/kevin/com/two95/assignment5/Car.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package com.two95.assignment5;
+
+public abstract class Car {
+	private String name;
+	private String colour;
+	private int enginePower;
+	private double milage;
+	public abstract void designEngine();
+	public abstract void designDoor();
```
Comment (saba83rish) [inline opt-may-2016/kevin/com/two95/assignment5/Car.java:9]: dont over complicate things

File: opt-may-2016/kevin/com/two95/assignment5/CarInterface.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package com.two95.assignment5;
+
+public interface CarInterface {
+	public static final String NoColor="";
+	public static final String RedColor="RED";
+	public static final String BlueColor="Blue";
+	public void honk();
+	public void go(int newVelocity);
```
Comment (saba83rish) [inline opt-may-2016/kevin/com/two95/assignment5/CarInterface.java:8]: good idea about passing velocity as an argument in the go..as it is a variable

**Comment by saba83rish** [comment]
where are the vehicle, truck , car wash and dmv classes. read the assignment question again.

**Comment by saba83rish** [comment]
i am gonna merge it temporarily..but your assignment is still not complete

**Comment by kevinsanthosh** [comment]
do you mean that there is no need of those classes or i have'nt written
those classes?

**Comment by kevinsanthosh** [comment]
my assignments not completed?
what more do i need to complete?
are u saying about the assignments on loose coupling?

**Comment by saba83rish** [comment]
what you have done so far is good, but where are the vehicle, truck , car wash and dmv classes. read the assignment question again.

**Pull request merged**